builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -639,12 +639,23 @@ local function create_change_setting_formspec(dialogdata)
 		-- Flags
 		formspec = table.concat(fields)
 				.. "checkbox[0.5," .. height - 0.6 .. ";cb_defaults;"
+				--[[~ "defaults" is the name of a mapgen noise
+				setting. This is a checkbox when editing a
```

Comment:
These are actually 'noise parameter flags', and are not necessarily mapgen-related, although they usually are currently.
The comment about 'smoother map' is probably better, and clearer, as 'smoother noise'.